internal/literals/literals.go

```diff
@@ -140,7 +140,7 @@ func Obfuscate(file *ast.File, info *types.Info, fset *token.FileSet, ignoreObj
 
 		case *ast.BasicLit:
 			switch cursor.Name() {
-			case "Values", "Rhs", "Value", "Args", "X", "Y", "Results":
+			case "Values", "Rhs", "Value", "Args", "X", "Y", "Results", "Elts":
```

Comment:
I didn't quite understand this switch statement when tackling this issue myself. I understand that you're switching on the field name of the parent AST node, but I don't understand how adding another case that returns early allows more strings to be obfuscated... maybe this should have a comment explaining it above?

Reply:
you misread the switch - we only return early on the default case, so this is a "do obfuscate these fields", not a "do not obfuscate these fields".